src/rust/engine/process_executor/src/main.rs

```diff
@@ -0,0 +1,48 @@
+extern crate process_executor;
```

Comment:
Are the unit tests good enough? maybe kill this file altogether if so.

Reply:
I'm definitely going to want a tool to play around with this on a terminal when we get to implementing this by RPC for remote execution.